### DISCUSSION TITLE
How to offer my products using SP API?

### DISCUSSION THREAD

**Discussion by rkiguti** [abuzuhri/Amazon-SP-API-CSharp#278] [Q&A]
I am trying to understand the API documentation but I am thinking it is very confuse.

I want to sell my products on Amazon, and I want to offer it. I know that Amazon use a different concept, that the products already registered and I need to say that I sell that product too. Something like this.

Which are the API methods that I will need? 

I read the documentation and I understand that I need to use the PutListingsItem. Is it right? 

How can I say the price and the quantity?

Do I need to use another Amazon API instead of Seller API?

**Comment by abuzuhri** [comment]
not easy to write desecration for those details here its better read all amazon documentation and then start using the API
